ita-social-projects/StreetCode
# Rewrite authorization

### How to review
1. Read the documentation: https://docs.google.com/document/d/1pwLxWxhI1eYfIG59oPWsHbOcm3QbejwY/
2. Setup local project;
3. Test `Auth` endpoints in Swagger: 
![image](https://github.com/ita-social-projects/StreetCode/assets/113620149/5fa7f38b-1bd6-4e4e-8704-8b73167a02ba)
P.S. Requst body and input data requirements are in 1documentation1
4. Check if autorization works: 
![image](https://github.com/ita-social-projects/StreetCode/assets/113620149/14ce8765-22cd-4add-af9f-067247aed392)
P.S. after successfill authorization Lock near endpoints will bocome `'locked'`: 
![image](https://github.com/ita-social-projects/StreetCode/assets/113620149/b1c749d4-7ede-4578-a3f8-7ed608add754)
5. Go to endpoints, thar requre authorization and check if everything wirks okay( endpoints, that requires authrization are listed in `documentation`)
6. Setup database for `integration tests` and run `integration tests`;
7. Check code style;
